docs/cli/usage/customcf.md

```diff
@@ -1,18 +1,24 @@
 ---
-title: Custom AWS Resources
+title: Add custom AWS resources
```

Comment:
"Custom CloudFormation Stacks"? Might be easier to search for that.

Reply:
Custom CloudFormation stacks was the title in a prior version. I think this title is more descriptive for folks who are new to CFN. The content has the term "Custom CloudFormation stack" quite often, so I don't think SEO is going to be a problem.